frontend/content/posts/aws-cdk-structure-components/index.yml

```diff
@@ -0,0 +1,12 @@
+title: "A better way to structure AWS CDK projects around Nested Stacks"
+excerpt: "Work around CloudFormation limitation using NestedStack, and at the same time, keeping the logic of the component!."
+banner: header.png
+authorIds:
+  - rzamana
+url: https://www.sentiatechblog.com/aws-cdk-structure-components
+createdAt: "2020-02-26 12:00"
```

Comment:
It's sorted by that date, and more meant to be something like a db timestamp. If you want it to show up on the landing page, you could adjust it to today. Otherwise, I can also just merge. Up to you.

Reply:
I added the time where this post went online... Next time will be up-to-date! ;)